src/index.html

```diff
@@ -16,5 +16,10 @@
             </ul>
         </nav>
     </header>
+
+    <footer class="f-container">
+        <p class="footer-details">Created by Microverse under CC lincense</p>
+    </footer>
+
```

Comment:
Please try to add style for your footer: 
Your design : 
![image](https://user-images.githubusercontent.com/50754458/203110425-040f29e3-d7aa-4672-bd24-0d7287f2e018.png)

Expected design : 


<img width="551" alt="123557345-c082eb80-d790-11eb-969a-2fc82bbec349" src="https://user-images.githubusercontent.com/50754458/203110560-7b1e3292-3a1a-49f0-b8fb-0affd42a3323.png">